sandboxed_containers/upgrade-sandboxed-containers.adoc

```diff
@@ -8,6 +8,21 @@ toc::[]
 
 You can upgrade the components of {sandboxed-containers-first} by upgrading {sandboxed-containers-operator} and {sandboxed-containers-first} artifacts.
 
+[IMPORTANT]
+====
+When upgrading OpenShift Container Platform 4.10 with OpenShift sandboxed containers 1.2 to OpenShift Container Platform 4.11 with OpenShift sandboxed containers 1.3, the recommended upgrade path is as follows:
```

Comment:
Can remove the text "with OpenShift sandboxed containers" to #1: shorten the sentence and #2: avoid redundancy as the first bullet point talks about the upgrade versions.